<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseInput</class>
 <widget class="QDialog" name="DatabaseInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Setup</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Hi there! Thanks for using Jotel. To continue using the this program, please 
enter your database details below</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>232</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="databaseIPLabel">
      <property name="text">
       <string>Database IP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="databaseIPLineEdit">
      <property name="placeholderText">
       <string>localhost</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="databaseNameLabel">
      <property name="text">
       <string>Database Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="databaseNameLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>jotel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="databaseUserLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="databaseUserLineEdit">
      <property name="placeholderText">
       <string>admin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="databasePasswordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="databasePasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="portLabel">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="portLineEdit">
      <property name="placeholderText">
       <string>3306</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatabaseInput</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatabaseInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
